playground/infrastructure/helper.py

```diff
@@ -268,25 +270,28 @@ def _validate(tag: dict, supported_categories: List[str]) -> bool:
       In case tag is not valid, False
   """
   valid = True
+  tag_fields = {f.default for f in fields(TagFields)
+                if f.default not in
+                {o_f.default for o_f in fields(OptionalTagFields)}}
```

Comment:
Could you please name those variables more explicitly?

Reply:
Done.